vendors/ti/boards/cc3220_launchpad/aws_tests/config_files/aws_test_ota_config.h

```diff
@@ -68,6 +68,30 @@
  */
 #include "aws_test_ota_pal_rsa_sha1_signature.h"
 
+/**
+ * @brief Major version of the firmware.
+ *
+ *        This is used in the OTA demo to set the appFirmwareVersion variable
+ *        that is declared in the ota_appversion32.h file in the OTA library.
+ */
+#define APP_VERSION_MAJOR    0
+
+/**
+ * @brief Minor version of the firmware.
+ *
+ *        This is used in the OTA demo to set the appFirmwareVersion variable
+ *        that is declared in the ota_appversion32.h file in the OTA library.
+ */
+#define APP_VERSION_MINOR    9
+
+/**
+ * @brief Build version of the firmware.
+ *
+ *        This is used in the OTA demo to set the appFirmwareVersion variable
+ *        that is declared in the ota_appversion32.h file in the OTA library.
+ */
+#define APP_VERSION_BUILD    0
```

Comment:
0.9.2 as per comment above

Reply:
Updated in https://github.com/aws/amazon-freertos/pull/3179/commits/1a91157fe268d8068568eebd9fcd04a98e7cd81a